src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -59,6 +63,7 @@
  *
  * @author Oliver Burn
  */
+// -@cs[ClassFanOutComplexity] Number of classes current class relies on exceeds 25.
```

Comment:
Suppression comment should be on why you can't comply with check, not what violation was.
Example: https://github.com/checkstyle/checkstyle/blob/master/config/suppressions.xml#L74-L75